druid-shell/src/error.rs

```diff
@@ -24,6 +24,8 @@ use crate::platform::error as platform;
 pub enum Error {
     /// The Application instance has already been created.
     ApplicationAlreadyExists,
+    /// Tried to use the application after it had been dropped.
+    ApplicationDropped,
```

Comment:
The behavior ok the wayland backend is to only hold weak handles to the application in the windows, so that the application might be dropped and then the window used. Not sure if this is correct.